fastlane/Fastfile

```diff
@@ -335,16 +335,6 @@ platform :android do
       properties: gradleProperties,
       project_dir: 'android'
     )
-
-    unless is_snapshot_version?(version)
```

Comment:
Hmm for PHC it's not an issue since we don't have snapshots but once we update the plugin in `purchases-android` we will have to think about how to avoid releasing snapshots unnecessarily.

Reply:
I thought doing `closeAndReleaseRepository` on a SNAPSHOT didn't have any effect and this check was just to save time, but maybe we should confirm it haha